src/cloud/components/experiments/variants/CustomerSuccessLinkPanel.tsx

```diff
@@ -0,0 +1,33 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {
+  Panel,
+  LinkButton,
+  ComponentColor,
+  ComponentSize,
+  ButtonShape,
+} from '@influxdata/clockface'
+
+export const CustomerSuccessLinkPanel: FC<{}> = () => {
+  return (
+    <Panel>
+      <Panel.Header>
+        <h4>Need help getting started?</h4>
+      </Panel.Header>
+      <Panel.Body>
+        <LinkButton
+          style={{textAlign: 'center'}}
+          className="home-page--contact-button"
+          color={ComponentColor.Primary}
+          shape={ButtonShape.Default}
+          size={ComponentSize.Small}
+          text="Speak with an Expert"
+          href="https://calendly.com/c/EDCUW4IQ7Z4ZWYSB"
```

Comment:
Should we put the name of the location of the test into the Calendly Event Header so that when it gets booked into their calendars the onboarding team can see the source? Ror example: Speak with InfluxDB Cloud Expert (Load Data)

Reply:
There is now a seperate event for home page vs load data page.